lib/fastlane/plugin/wpmreleasetoolkit/helper/an_metadata_update_helper.rb

```diff
@@ -137,12 +137,16 @@ def initialize(block_key, content_file_path, release_version)
 
       def is_handler_for(key)
         values = key.split('_')
-        key.start_with?(@rel_note_key) && values.length == 4 && (Integer(values[3].sub(/^[0]*/, '')) != nil rescue false)
+        key.start_with?(@rel_note_key) && values.length == 4 && is_int?(values[3].sub(/^[0]*/, ''))
       end
 
       def generate_block(fw)
         super(fw) unless File.zero?(@content_file_path)
       end
+
+      def is_int?(value)
```

Comment:
This looks ok to me – functionally identical to what came before.

The conditionals that reference this though IMHO could do with some cleanup and documentation though in the future 😅

Reply:
Yeah, lots of ternary operator shenanigans I agree. I plan to tackle that probably during #203.